_translator-files/po/de/Tips-Tricks-More.po

```diff
@@ -45,116 +47,118 @@ msgid ""
-msgstr ""
+msgstr "Du kannst auch [VoiceMeeter](https://www.vb-audio.com/Voicemeeter/banana.htm) (Banana) für Windows oder [BlackHole](https://github.com/ExistentialAudio/BlackHole) für macOS verwenden, um die Jamulus-Ausgabe an mehrere Ziele zu leiten, z. B. an deine Kopfhörer und die Meeting-Anwendung gleichzeitig."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:37
 msgid "### Recording Jamulus on Windows with Reaper"
-msgstr ""
+msgstr "### Aufnahme von Jamulus unter Windows mit Reaper"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:39
 msgid "Jamulus user [Rob Durkin](https://sourceforge.net/u/bentwrench/profile/) has written a [guide to recording the output of Jamulus](https://docs.google.com/document/d/1tENfNKTWHasuTg33OdLLEo4-OOhWJolOo42ffSARxhY/edit) (Google Doc) using the ReaRoute add-on for [Reaper](https://www.reaper.fm/)."
-msgstr ""
+msgstr "Jamulus-Benutzer [Rob Durkin](https://sourceforge.net/u/bentwrench/profile/) hat eine [Anleitung zur Aufzeichnung der Jamulus-Ausgabe](https://docs.google.com/document/d/1tENfNKTWHasuTg33OdLLEo4-OOhWJolOo42ffSARxhY/edit) (Google Doc) unter Verwendung des ReaRoute-Add-ons für [Reaper](https://www.reaper.fm/) geschrieben."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:41
 msgid "### Sharing song/chord sheets"
-msgstr ""
+msgstr "### Gemeinsame Nutzung von Lied-/Akkordblättern"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:43
 msgid "Jamulus user [BTDT](https://sourceforge.net/u/btdt/profile/) has written a system called [305keepers](https://github.com/keepers305/Song-Sheet-Sharing-Web-Pages), a web application that allows a \"Jam leader\" to push song sheets (in PDF format) to \"Jammers\" in real time using standard web browsers."
-msgstr ""
+msgstr "Jamulus-Benutzer [BTDT](https://sourceforge.net/u/btdt/profile/) hat ein System namens [305keepers](https://github.com/keepers305/Song-Sheet-Sharing-Web-Pages) geschrieben, eine Webanwendung, mit der ein \"Jam-Leader\" Songblätter (im PDF-Format) in Echtzeit über Standard-Webbrowser an \"Jammer\" senden kann."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:45
 msgid "### Jamulus Client Linux start script"
-msgstr ""
+msgstr "### Jamulus Client Linux Startskript"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:47
 msgid "Here's a Linux start script for Jamulus using an old Audigy4 sound card, the large number of available audio faders for which makes it hard to get the correct settings."
-msgstr ""
+msgstr "Hier ist ein Linux-Startskript für Jamulus, das eine alte Audigy4-Soundkarte verwendet, deren große Anzahl an verfügbaren Audio-Fadern es schwierig macht, die richtigen Einstellungen zu finden."
```

Comment:
Should probably go to the Knowledge base

@gilgongo (sorry for pinging you multiple times, but I think it's just mailing you once).

Reply:
It's something Volker had on the original website, but I guess we should move it out to the KB since we can't really maintain it.

Reply:
Agree. Can you open an issue (and later remove it)?